scripts/quests/jeuno/Apocalypse_Nigh.lua

```diff
@@ -0,0 +1,290 @@
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 5 then
+                        return quest:progressEvent(10061)
+                    end
+                end,
+            },
+            ['Yin_Pocanakhu'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 5 then
+                        return quest:progressEvent(10060)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [10057] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 5)
+                    quest:setVar(player, 'Wait', getVanaMidnight())
+                end
+            },
+        },
+
+        [xi.zone.NORG] =
+        {
+            ['Gilgamesh'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 5 and os.time() >= quest:getVar(player, 'Wait') then
+                        return quest:progressEvent(232, 252)
+                    end
+                    if quest:getVar(player, 'Prog') == 6 and os.time() >= quest:getVar(player, 'Wait') then
+                        return quest:progressEvent(234, 252)
+                    end
+                end,
+            },
+
+            ['_700'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 5 and quest:getVar(player, 'Wait') > os.time() then
+                        return quest:event(235)
+                    end
+                end,
+            },
+
+            onEventUpdate =
+            {
+                [232] = function(player, csid, option)
+                    updateItemEvent(player, option)
+                end,
+                [234] = function(player, csid, option)
+                    updateItemEvent(player, option)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [232] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog', 6)
+                    if option >= 1 and option <= 4 then
+                        quest:setVar(player, 'Prog', 7)
+                        finishProgress(player, option)
+                    end
+                end,
+                [234] = function(player, csid, option, npc)
+                    if option >= 1 and option <= 4 then
+                        quest:setVar(player, 'Prog', 7)
+                        finishProgress(player, option)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_COMPLETED
+        end,
+
+        [xi.zone.NORG] =
+        {
+            ['Gilgamesh'] =
+            {
+                onTrigger = function(player, npc)
+                    quest:event(233)
```

Comment:
This should be returned.  All onTrigger interactions need to return something in the quest class, else the default action will be hit.